src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -0,0 +1,50 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.Order;
+import mate.academy.spring.model.ShoppingCart;
+import mate.academy.spring.model.dto.response.OrderResponseDto;
+import mate.academy.spring.service.OrderService;
+import mate.academy.spring.service.ShoppingCartService;
+import mate.academy.spring.service.UserService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+    private final OrderService orderService;
+    private final ShoppingCartService shoppingCartService;
+    private final UserService userService;
+    private final DtoResponseMapper<OrderResponseDto, Order> orderDtoResponseMapper;
+
+    public OrderController(OrderService orderService,
+                           ShoppingCartService shoppingCartService,
+                           UserService userService,
+                           DtoResponseMapper<OrderResponseDto, Order> orderDtoResponseMapper) {
+        this.orderService = orderService;
+        this.shoppingCartService = shoppingCartService;
+        this.userService = userService;
+        this.orderDtoResponseMapper = orderDtoResponseMapper;
+    }
+
+    @PostMapping("/complete/{userId}")
+    public OrderResponseDto create(@PathVariable Long userId) {
+        ShoppingCart shoppingCart = shoppingCartService.getByUser(userService.get(userId));
+        Order order = orderService.completeOrder(shoppingCart);
+        return orderDtoResponseMapper.toDto(order);
+    }
+
+    @GetMapping
+    public List<OrderResponseDto> getAllOrders(@RequestParam Long userId) {
```

Comment:
You should or USE or DO NOT USE **Order** in the method names
createOrder() and getAllOrders()
OR
create() and getAll()